uktrade/data-hub-frontend
# Release new card styling to all users

## Description of change

We want to release the new Card stylings to all users.

These cards are for Data Hub activities (Interaction, Service Delivery, Investment Project, OMIS Order, Company Referral) and external activity (Companies House Accounts, Companies House Company, HMRC Exporter). This is for the company activity feed and contact activity feed.

This styling work was carried out in #4854 

We also removed the 'Show details for all activities' checkbox on the company activity feed as this is now redundant.

## Test instructions

-Make sure your `user-contact-activities` feature flag is **off** 
-Check out this branch
-Visit a contact's activity feed, [Joe Doe](http://localhost:3000/contacts/68be55fc-544d-4d9c-a264-977b917f1fd3/interactions?page=1) is a good example: 
-You should now see the new card stylings 
-Visit a company's activity feed, [One List Corp](http://localhost:3000/companies/375094ac-f79a-43e5-9c88-059a7caa17f0/activity) is a good example
-You should also see the new stylings here

## Screenshots

### Before

Contact activity feed
![image](https://user-images.githubusercontent.com/83657534/188863689-cb84b791-b013-4dea-97c7-69aae6d19e83.png)

Company activity feed
![image](https://user-images.githubusercontent.com/83657534/188864551-91ebdbca-a0aa-4bbe-9e03-e0cf7290c85d.png)


### After

Contact activity feed
![image](https://user-images.githubusercontent.com/83657534/188863868-792270a2-63e3-49a0-bbdc-b89a865c470a.png)

Company activity feed
![image](https://user-images.githubusercontent.com/83657534/189372136-507005fa-7361-41b7-b617-eacd1ced2264.png)


## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
